contents/handbook/content-and-docs/docs.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Docs ownerhship
+sidebar: Handbook
+showTitle: true
+---
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
+  - Correcting mistakes reported by users.
+  - Clarifying documentation where needed based on support tickets.
+  - Ensuring public betas have documentation which is linked to from feature preview menu
+
+- **Content marketing** is responsible for improving the docs. This means:
+  - Reviewing and improving draft documentation created by product teams.
+  - Identifying and improving low quality documentation.
+  - Ensuring screenshots and other visual elements are up-to-date.
+  - Shipping supplementary docs and tutorials based on feedback and emerging use cases.
+
+- **Website & Docs** is responsible for design, organization, and discovery. This means:
+  - The design and content of index pages.
+  - The overall layout of docs and how they're organized.
+  - In-page elements and components – e.g. light and dark mode screenshots.
+  - Website search and other elements that help users find the answers they need.
+
+## Ownership in Content & Docs team
+
+We aim to have a named individual for most areas of the docs, but this doesn't preclude anyone working on any part of the docs they deem necessary.
```

Comment:
```suggestion
We aim to have a named individual for most areas of the docs. Docs owners are responsible for:

- Reviewing and polishing docs created by product teams 
- Ongoing maintenance of those docs (e.g. updating as product changes)
- Improving docs based on user feedback / suggestions from the support team.

This doesn't preclude anyone from submitting new or improved docs for any product – remember: PRs > issues
```

@Lior539 Adding a bit more context based on your feedback. It wasn't very clear.